lrlex/src/lib/parser.rs

```diff
@@ -340,6 +340,7 @@ where
         let mut grmtools_section_span_map = HashMap::new();
         let mut grmtools_section_lex_flags = UNSPECIFIED_LEX_FLAGS;
         if let Some(j) = self.lookahead_is("%grmtools", i) {
+            // lrlex currently doesn't have any `key: value' settings.
```

Comment:
This is actually wrong, I totally forgot that `regex` defined some limit related values.